Runtime/AvatarCreator/WebRequests/AuthAPIRequests.cs

```diff
@@ -9,20 +9,21 @@ public class AuthAPIRequests
     {
         private readonly string domain;
         private readonly IDictionary<string, string> headers = CommonHeaders.GetHeadersWithAppId();
+        private readonly string rpmAuthBaseUrl;
 
         private readonly WebRequestDispatcher webRequestDispatcher;
 
         public AuthAPIRequests(string domain)
```

Comment:
Is this domain = subdomain? and if so why is it not using core settings subdomain that is used everywhere else?

Reply:
It seems to be initialized with that value (see screenshot below):

I can update it if you think it makes sense to.

![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/121796327/94854abe-75ab-4478-9da6-e4c9b8bd047c)